netlify.toml

```diff
@@ -0,0 +1,3 @@
+[template.environment]
+  CONTENTFUL_SPACE_ID = ""
+  CONTENTFUL_ACCESS_TOKEN = ""
```

Comment:
Should we also add the `incoming-hook`??
https://docs.netlify.com/site-deploys/create-deploys/#template-configuration

Reply:
Good call, done.